test/verify/check-login

```diff
@@ -166,6 +166,74 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
 
         self.allow_restart_journal_messages()
 
+
+    @skipImage("logs in via ssh, not cockpit-session", "fedora-coreos")
+    def testLogging(self):
+        m = self.machine
+        b = self.browser
+
+        def assert_messages(has_last, n_fail):
+            if has_last:
+                b.wait_present('#login-messages')
+                b.wait_present('#last-login')
+                b.wait_in_text('#last-login', "Last login")
+
+                if n_fail:
+                    b.wait_present('#last-failed-login')
+                    b.wait_in_text('#login-messages', 'There were {} failed'.format(n_fail))
+
+            else:
+                assert not n_fail
+                b.wait_present('#login-messages[empty=yes]')
+                self.assertFalse(b.is_present('#last-login'))
+                self.assertFalse(b.is_present('#last-failed-login'))
+
+        def verify_correct(has_last, n_fail):
+            b.login_and_go('/system')
+            assert_messages(has_last, n_fail)
+
+            # reload and make sure it's still there (or not)
+            b.reload()
+            b.enter_page('/system')
+            assert_messages(has_last, n_fail)
+
+            if has_last:
+                # dismiss and make sure it's gone
+                b.click('#login-messages button')
+                assert_messages(False, 0)
+
+                # reload and make sure it's still gone
+                b.reload()
+                b.enter_page('/system')
+                assert_messages(False, 0)
+
+            b.logout()
+
+        m.start_cockpit()
+
+        # Clean out the relevant logfiles
+        m.execute("truncate -s0 /var/log/{[bw]tmp,lastlog} /var/run/utmp")
```

Comment:
This test is nondestructive, so it would be nice to back up these files (except for the transient /run/utmp) and restore them with `addCleanup()`. PR #13751 introduces a convenient `self.restore_file()` for that purpose, but that hasn't  landed yet. But unless you are interested in learning how this works, I'm fine to just add this as a task to #13717 as it's not that important.

Reply:
I'll look into this.

Reply:
As per our conversation yesterday, this isn't important.